pkg/sources/github/github_test.go

```diff
@@ -742,8 +768,6 @@ func TestGetGistID(t *testing.T) {
 	}{
 		{[]string{"https://gist.github.com", "12345"}, "12345"},
 		{[]string{"https://gist.github.com", "owner", "12345"}, "12345"},
-		{[]string{"https://gist.github.com"}, ""},
```

Comment:
Technically these should never occur because there's an explicit length check earlier.

https://github.com/trufflesecurity/trufflehog/blob/9d4cf87c020a38f479daf6dab00d7ab970f13b35/pkg/sources/github/github.go#L1114-L1116

https://github.com/trufflesecurity/trufflehog/pull/2379#discussion_r1496920923